modules/simple-payments/simple-payments.css

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
Should we add this to the concatenation in class.jetpack.php? That's a non-blocker for merge, but should be done or issued so we can get it in there.